internal/server/handler.go

```diff
@@ -309,14 +308,18 @@ func (s *Server) BulkFindEntities(
 	return recon.BulkFindEntities(ctx, in, s.store, s.mapsClient)
 }
 
-// Import implements API for Mixer.Import
-func (s *Server) Import(
-	ctx context.Context, in *pb.ImportRequest,
-) (*pb.ImportResponse, error) {
-	cache, err := data.Import(ctx, in, s.store)
-	if err != nil {
+// UpdateCache implements API for Mixer.UpdateCache
+func (s *Server) UpdateCache(
+	ctx context.Context, in *pb.UpdateCacheRequest,
+) (*pb.UpdateCacheResponse, error) {
+	if err := s.cachedata.UpdateSVGCache(ctx, s.store); err != nil {
```

Comment:
Consider encapsulating this logic in a single Cache.UpdateCache(...) call vs the API layer implementing it.

Reply:
Done